base/osutils.jl

```diff
@@ -56,11 +56,3 @@ end
 @unix_only windows_version() = (0,0)
 
 WINDOWS_XP_VER = (5,1)
```

Comment:
Oh, no kidding, guess my wine thinks it's either server 2003 or xp 64. I don't have either of those handy aside from in Wine, so I think comparing to vista does make sense. Will change that.